modules/tensor_mechanics/src/materials/ADLAROMANCEStressUpdateBase.C

```diff
@@ -692,10 +697,11 @@ ADLAROMANCEStressUpdateBase::convertOutput(const std::vector<Real> & old_input_v
     return 0.0;
 
   ADReal expout = std::exp(rom_output);
```

Comment:
This is not never 0.0! E.g.`std::exp(-800.0) == 0.0`

Reply:
? rom_output isn't necessarily zero or negative...

Reply:
We divide by expout below without further checks. I suggest adding an assertion to make sure expout > 0

Reply:
Oh, I see rom_output is guaranteed to be positive?

Reply:
Uh....I'm not sure it's guaranteed. Maybe. Certainly an assert is easy to add

Reply:
Assert added